### PR TITLE
fix(argo-cd): Make the PathType configurable when using single ingress resource in AWS

### PR DESCRIPTION
**Pull request by w32-blaster**
Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

### CODE REVIEW

File: charts/argo-cd/templates/argocd-server/ingress.yaml [hunk 1]
```diff
@@ -39,7 +39,7 @@ spec:
           {{- range $p := $paths }}
           {{- if and $.Values.server.ingressGrpc.isAWSALB $.Values.server.ingressGrpc.enabled }}
           - path: {{ $p }}
-            pathType: Prefix
+            pathType: {{ .Values.server.ingressGrpc.pathType }}
```
[the remark this answers is not in the document]
Comment (w32-blaster) [reply charts/argo-cd/templates/argocd-server/ingress.yaml]: thank you. Fixed

**Review by mkilchhofer — APPROVED** [review]

**Review by yu-croco — APPROVED** [review]
Thank you for your contribution. LGTM.

**Comment by w32-blaster** [comment]
Thanks for merging it. Any idea how long it takes to have the new release available?

**Comment by yu-croco** [comment]
@w32-blaster 
Merge triggers publishing the new version, so it's is already available.
https://artifacthub.io/packages/helm/argo/argo-cd?modal=changelog

**Pull request merged**